etcdmgmt/etcd-mgmt.go

```diff
@@ -0,0 +1,73 @@
+package etcdmgmt
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func StartEtcd() error {
+
+	log.Info("Starting etcd")
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Could not able to get hostname")
+		return err
+	}
+
+	listenClientUrls := "http://" + hostname + ":2379"
+
+	advClientUrls := "http://" + hostname + ":2380"
+
+	etcdStart := exec.Command("/bin/etcd",
```

Comment:
@gaurav36 : etcdCommand interprets what command string you have formed for the etcd instance to bring up. If you look at the signature of exec.Command it returns a command structure.  Check other projects if you wish, mostly they use 'cmd' for the return value.